docs/tutorials/text-classification-tutorial.mdx

```diff
@@ -0,0 +1,295 @@
+---
+title: How MindsDB & Hugging Face Can Help Stop Hate-Speech
+sidebarTitle: Use MindsDB's, Hugging Face and MySQL Datasource to Perform NLP Queries that Identify Hate Speech in Your Dataset
+---
+
+## Introduction
+
+In today's world, digital conversation is king. Whether it be text message, email, or social media, we do the majority of our communicating in a virtual setting. With all this information spreading around, negative language is bound to start showing up. 
+
+Hate speech is defined as, 'abusive or threatening speech or writing that expresses prejudice based on ethnicity, religion, sexual orientation, or similar grounds'. This offensive language comes in many forms, including memes, messages, social posts, gestures, and icons (even emojis can be used the wrong way these days).
```

Comment:
The paragraph explaining hate speech is not necessary. Please keep it like this:

> negative language, also known as hate speech, is bound to start showing up.

Reply:
is this better?